pages/mint-calendar/[launchpad_id]/project/[project_id].js

```diff
@@ -159,6 +238,11 @@ const ProjectPage = ({ project }) => {
 									)}
 								</>
 							)}
+							{showRemindMe && data && data.status === 'upcoming' && !isEndedComing ? (
+								<ReminderCheckbox value={remindMe} onChange={() => changeRemindMe()} />
+							) : (
+								<></>
+							)}
```

Comment:
fyi conditionalnya bisa gini sih, sama props `onChange`nya ga perlu di wrap pake anonymous function lagi
```suggestion
							{showRemindMe && data && data.status === 'upcoming' && !isEndedComing && (
								<ReminderCheckbox value={remindMe} onChange={changeRemindMe} />
							)}
```

Reply:
ok